README.md

```diff
@@ -27,10 +27,18 @@ The Helium Miner with the required configuration files added.
 
 https://github.com/NebraLtd/hm-miner
 
-## eccprog
+## Secure Element Provisioning
```

Comment:
Similarly to above the idea is that it stays the same name as the container, but as I failed to update should be gwmfr